augur/data/schema-multiplot.json

```diff
@@ -0,0 +1,126 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://nextstrain.org/schemas/multiplot/v1",
+    "title": "Nextstrain multiplot data schema proposal",
+    "type": "object",
+    "additionalProperties": false,
+    "required": ["config", "collections"],
+    "properties": {
+        "config": {
+            "description": "Config for display text and display options",
+            "type": "object",
+            "additionalProperties": false,
+            "required": ["collections", "groupings"],
+            "properties": {
+                "collections": {
```

Comment:
I prefer "measurements" or "experiments" over "collections", but we can continue to think about this in the background as the feature is developed.

Reply:
I really like [your proposal above](https://github.com/nextstrain/augur/pull/827#discussion_r781510047) where measurements are the array of (ahem) measurements within a collection

Reply:
Hah, yeah "measurements" made more sense once it was within the "collection" 😆